workflow/resources/datavzrd/units-template.yaml

```
@@ -0,0 +1,15 @@
+__use_yte__: true
+
+name: ?f"Inputs used to start the pipeline"
```

This could be a bit more specific to the `units.tsv`, maybe:
```suggestion
name: ?f"Definition of input files and technical sequencing parameters"
```